frontend/src/components/Profile/BasicTabs.tsx

```diff
@@ -1,8 +1,11 @@
 import * as React from 'react';
-import { Tabs, Tab, Typography, Box } from '@mui/material';
+// import { useState } from 'react';
+import { Tabs, Tab, Typography, Box, TextField } from '@mui/material';
+import { Controller, Control } from 'react-hook-form';
+import { UserForm } from './types';
 
 interface TabPanelProps {
-  children?: React.ReactNode;
+  children?: any;
```

Comment:
maybe we change any for another type?  i know that hard)

Reply:
i create frontend part write feedback on my pr